File: FIbonacciNumber.js
let pre2 =0;
let pre1=1;
console.log(pre2);
console.log(pre1);
for(let i=0;i< 18;i++){
    let newFibo =pre2+pre1;
    console.log(newFibo);
    pre2=pre1;
    pre1=newFibo;
}

File: FibonacciNumber.java
public class FibonacciNumber{
public static void main(String[] args) {
    int pre2 =0;
    int pre1 =1;
    System.out.println(pre2);
    System.out.println(pre1);
    for(int i=0;i< 18;i++){
        int newFibo = pre2 +pre1;
        System.out.println(newFibo);
        pre2=pre1;
        pre1=newFibo;
    }
}
}
